Graphics/GraphicsEngineWebGPU/include/TextureViewWebGPUImpl.hpp

```diff
@@ -61,6 +61,10 @@ class TextureViewWebGPUImpl final : public TextureViewBase<EngineWebGPUImplTrait
 
     WGPUTextureView GetMipLevelSRV(Uint32 Mip);
 
+    WGPUTextureView GetMipLevelRTV(Uint32 Slice, Uint32 Mip);
+
+    WGPUTextureView GetMipLevelSRV(Uint32 Slice, Uint32 Mip);
+
 private:
     WebGPUTextureViewWrapper              m_wgpuTextureView;
     std::vector<WebGPUTextureViewWrapper> m_wgpuTextureMipSRVs;
```

Comment:
These views should be in the Texture object

Reply:
I specifically asked about this when we first started working on it, and we decided that the logic would be like in D3D12